### DISCUSSION TITLE
Having  upload issues with large or many files

### DISCUSSION THREAD

**Discussion by KageKeeper** [nextcloud/all-in-one#686] [Questions] [answered]
When I am attempting to upload a large number of files, or large-ish video files, Nexcloud ends  up choking on it. I lose connection completely, and the only way that I have been able to 'fix' it is to shutdown the containers and bring them back up. I also get some pm.max_children errors in the logs

I know there are some variable that can be set in the php.ini, and I believe the proper place for pm is in /usr/local/etc/php-fpm.d/www.conf. However, this is not persistent, and I am unsure how to reload the configs without restarting the containers.

Is there a work around for this? I would really like to stop having these upload problems. :)

Thank you.

**Comment by szaimen** [comment, marked as answer]
Can you send the logs?

**Reply by KageKeeper** [reply]
Well, I would love too. However, it seems I am no longer having the issue now that I posted about it. I have uploaded multiple batches of pictures and videos and it worked with zero issue. So, thank you, I guess? lol
